### ISSUE TITLE
Grunt build error

### ISSUE THREAD

**Issue by damienstanton** [jdkolassa/Oeuvre#4]
When trying to build, we get: 

``` bash
Running "nodewebkit:src" (nodewebkit) task 
Fatal error: No files matching
```

**Comment by jdkolassa** [comment]
Yeah, I had the same problem this weekend. I don't understand it at all. I guess I'll just do it manually. 

Sent via my Samsung Galaxy S5. Any typos are the phone's fault. 

-------- Original Message --------

**Comment by damienstanton** [comment]
@jdkolassa, see my fix on issue #5. Feel free to merge if it works for you now.
